apps/advisor/pkg/app/checkscheduler/checkscheduler.go

```diff
@@ -176,3 +188,33 @@ func (r *Runner) cleanupChecks(ctx context.Context) error {
 
 	return nil
 }
+
+func getEvaluationInterval(pluginConfig map[string]string) (time.Duration, error) {
+	// Default value
+	evaluationInterval := 24 * time.Hour
+
+	configEvaluationInterval, ok := pluginConfig["evaluation_interval"]
+	if ok {
+		var err error
+		evaluationInterval, err = gtime.ParseDuration(configEvaluationInterval)
+		if err != nil {
+			return 0, fmt.Errorf("invalid evaluation interval: %w", err)
+		}
+	}
+	return evaluationInterval, nil
+}
+
+func getMaxHistory(pluginConfig map[string]string) (int, error) {
+	// Default value
+	maxHistory := 10
```

Comment:
Maybe make `maxHistory` and `evaluationInterval` constants which you can then also reference in the test?